toolchain/check/import_ref.cpp

```diff
@@ -2126,7 +2126,8 @@ static auto MakeImplDeclaration(ImportContext& context,
        {.self_id = SemIR::TypeInstId::None,
         .constraint_id = SemIR::TypeInstId::None,
         .interface = SemIR::SpecificInterface::None,
-        .witness_id = witness_id}});
+        .witness_id = witness_id,
+        .is_final = import_impl.is_final}});
```

Comment:
I'm surprised we don't get a warning / clang-tidy failure for missing this initializer.

Reply:
Yeah, it does not have a default value, so we're supposed to.